hugegraph-rocksdb/src/main/java/com/baidu/hugegraph/backend/store/rocksdb/RocksDBStdSessions.java

```diff
@@ -247,6 +251,36 @@ public synchronized void dropTable(String... tables)
         }
     }
 
+    public void reload() throws RocksDBException {
```

Comment:
unneeded to reload rocksdb if not operate batch after CF is dropped.
but we can keep these code for future use (don't --amend)